src/main/resources/hibernate.cfg.xml

```diff
@@ -6,13 +6,17 @@
 <hibernate-configuration>
     <session-factory>
         <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
-        <property name="connection.url">jdbc:mysql://localhost/cinema?serverTimezone=UTC</property>
+        <property name="connection.url">jdbc:mysql://localhost/my_social_network?serverTimezone=UTC</property>
         <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
-        <property name="connection.username">ma_admin</property>
-        <property name="connection.password">1234</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">root</property>
```

Comment:
Everything looks good just reset these settings because we don't have to change them